src/htmlParser.js

```diff
@@ -39,12 +39,16 @@ class HtmlParser {
 
       const contentText = comment.contentText.runs
 
+      const customEmojis = []
       contentText.forEach((content) => {
         if (content.text.trim() === '') {
           text = text + '<br>'
         } else {
           text = text + content.text
         }
+        if (typeof content.emoji !== 'undefined') {
+          customEmojis.push({"text":content.text,"emojiThumbnails":content.emoji.image.thumbnails})
```

Comment:
```suggestion
          customEmojis.push({
            text:            content.text,
            emojiThumbnails: content.emoji.image.thumbnails
          })
```

Reply:
The space between "text:" and "context.text" doesn't match the rest of the code

Reply:
Ya but does that make it easier to read?